### ISSUE TITLE
Quick Discover Fix

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-core#1016]
The Discover gallery does not show all featured projects. Several of us tried to track down the root of this "bug" (#1010), but it is not a bug after all. 😕 

**Cause**: Part of the initial strategy was to promote local content, which is great. So if a user in a supported locale views Discover, they only see projects from other users in that locale. I assumed this was a goal, not a launch feature. My mistake.

**Problem**: A fledgling app without any content needs to fill its empty shelves before all else. Users in es-MX (a supported locale), still see zero projects in Discover. Users in other supported locales will not see any of the helper how-to projects I created. Users in unsupported locales actually have a better view than the rest of us.

Unfortunately, this likely had some impact on post-launch performance. Fortunately, we're ready to launch the work completed in January. I believe @gvn is interested in building the release (and sharing the process with others who are interested in learning).

As a quick fix, I recommend we strip `lang` from our Discover route. I believe it sits here: https://github.com/mozilla/webmaker-core/blob/develop/src/components/project-list/project-list.jsx#L79. What do you think? A more thorough fix would enable us to feature a project in one or all locales, but this would require more code and more moderation than we can probably muster. Add your thoughts below.

It will be interesting to see the impact of this quick fix coupled with the introduction hashtags.

@vazquez @alanmoo @cadecairos @ryanwse

**Comment by ryanwarsaw** [comment]
What are your thoughts on providing a user toggle-able option that allows them to choose whether they want to only see localized content or not? Surely we would need to build such a view in the future anyways (for other user related options).

**Comment by xmatthewx** [comment]
@ryanwse – you're right, that could be an ideal solution. We just don't have the time to design, build, test, and localize it. Adding a toggle is relatively easy. Figuring out where to put it, how to label it, and what to call it gets more complicated. Would sketch up a proposal to consider for later. In the meantime, Hashtags will also give people a chance to create a connection between local content.

**Comment by xmatthewx** [comment]
As a quick demo of the problem, hit both of these requests. You'll see that the english route doesn't return project ID 56558.

https://api.webmaker.org/discover/fr-US?page=1&count=50
https://api.webmaker.org/discover/en-US?page=1&count=50
https://api.webmaker.org/discover/?page=1&count=50

see: [line 79](https://github.com/mozilla/webmaker-core/blob/develop/src/components/project-list/project-list.jsx#L79) and [line 13](https://github.com/mozilla/webmaker-core/blob/develop/src/components/project-list/project-list.jsx#L13) in the project-list component.

**Comment by cadecairos** [comment]
Project 56558 shouldn't show up in the en-US route - it's lang is set to `fr`

edit - so is the problem is that you can't see a non-localized set of discover projects without changing the system locale?

**Comment by xmatthewx** [comment]
You are right that it's not a bug in the code. We don't have enough curators to feature activity, so discover is likely to feel stagnant. And, helper and how-to projects  haven't been localized, so others don't see them.
